pac.go

```diff
@@ -6,25 +6,36 @@ import (
 	"sync"
 
 	"github.com/getlantern/byteexec"
+	"github.com/getlantern/golog"
 )
 
-var mu sync.Mutex
-var be *byteexec.Exec
+var (
+	log = golog.LoggerFor("pac")
+
+	mu sync.Mutex
+	be *byteexec.Exec
+)
 
 // EnsureHelperToolPresent checks if helper tool exists and extracts it if not.
 // On Mac OS, it also checks and set the file's owner to root:wheel and the setuid bit,
 // it will request user to input password through a dialog to gain the rights to do so.
-// fullPath: the file to be checked and generated if not exists.
+// path: absolute or relative path of the file to be checked and generated if
+// not exists. Note - relative paths are resolved relative to the system-
+// specific folder for aplication resources.
 // prompt: the message to be shown on the dialog.
 // iconPath: the full path of the icon to be shown on the dialog.
-func EnsureHelperToolPresent(fullPath string, prompt string, iconFullPath string) (err error) {
+func EnsureHelperToolPresent(path string, prompt string, iconFullPath string) (err error) {
```

Comment:
No longer assume a full path.  This allows byteexec to place the binary in the system-dependent default location, which is what we want.